packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -253,3 +253,22 @@ If the middle initial is not empty, include it as part of the full name, using j
 
 If the user is a contractor and is a member of the "West Coast Users" user group, output "West coast contractors", else output "Others".<br>
 `user.profile.isContractor && user.isMemberOf({'group.profile.name': 'West Coast Users'}) ? "West coast contractors" : "Others"`
+
+## Security Expressions
+
+In [App Sign On Policies](/docs/reference/api/policy/#app-sign-on-policy), you can additionally specify certain [Rule Conditions](/docs/reference/api/policy/#conditions) using expressions based on the security context of the request.
+The security context can be used to match requests for a given risk level or for matching a given [Behavior](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
```

Comment:
nit: should we call Behavior -> User Behavior? Might be worth looking around the docs to see how it is named elsewhere